### DISCUSSION TITLE
Name format - just an idea :)

### DISCUSSION THREAD

**Discussion by LouisChoiseul** [AAndyProgram/SCrawler#203] [Ideas]
Hi, I don't know very deeply this project but I have a suggestion of this feature to implement that would be very helpful.
On each site setting could have an input field for the file formatted name, using some predefined "variables", like post_date, username, post_id and so on.

I know this other project called UltimaScrapper that has this feature and it's really great, take a look:
https://github.com/UltimaHoarder/UltimaScraper?tab=readme-ov-file#path_formatting

**Comment by AAndyProgram** [comment]
No, sorry... 🤷🏼‍♂️ 
You can use scripts https://github.com/AAndyProgram/SCrawler/wiki/Settings#how-to-use-the-script
